internal/project/service.go

```diff
@@ -222,3 +226,51 @@ func (s *ServiceImpl) deleteProjectVersion(ctx context.Context, version *Version
 
 	return nil
 }
+
+func (s *ServiceImpl) CreateLanguagesVersion(ctx context.Context, projectID int, name string) error {
```

Comment:
Should the errors in this function be more descriptive `using errors.Wrap()` or is just returning the error itself sufficient?

Reply:
I think the error itself is sufficient because it comes from different functions and every function have it's own error message

Reply:
I would think that wrapping it in an extra message instead of just returning the error, might allow us to easier pinpoint which of all the error checks in this function, is the one that is triggered.

While it might be more verbose this way, it does help whoever might need to investigate an error in the future.